study/src/org/labkey/study/model/ParticipantIdImportHelper.java

```diff
@@ -165,6 +165,8 @@ public String translateParticipantId(@Nullable Object p) throws ValidationExcept
                 participantId = p.toString();
         }
 
+        participantId = participantId != null ? participantId.trim() : null;

```

Comment:
trim to null? Is `participantId = null` functionally distinct from `participantId = ""`?
```suggestion
        participantId = StringUtils.trimToNull(participantId);
```

Reply:
yes, done.